style.css

```diff
@@ -413,7 +409,96 @@ body {
   font-weight: normal;
 }
 
-.btn-link {
-  border: 1px solid red;
-  float: left;
+/* contact form */
+
+.form-container {
+  background-color: #6070ff;
+  width: 100%;
+  height: 750px;
+  border: none;
+  left: 0;
+  top: 1rem;
+  border-top-left-radius: 60px;
+}
```

Comment:
- [ ] According to the project requirements, you need to use a CSS Flexbox property to layout the form elements, please add it.